samples/sample_raw_fault_injector/main.go

```diff
@@ -30,6 +31,9 @@ func main() {
 			&faultinjectors.FaultInjectorOptions{
 				// enable logging all traffic to a JSON file
 				JSONLFile: filepath.Join(*logs, "sample-rawfaultinjector-traffic.json"),
+				TransformerOptions: &logging.TransformerOptions{
+					ExcludePayloadData: true,
```

Comment:
I would think we wouldn't do this in our example. I think people would run this expecting to see all the data. What do you think?

Reply:
makes sense - removing this